setup.py

```diff
@@ -109,7 +109,7 @@ def get_extras_require() -> Dict[str, List[str]]:
             "torchvision==0.8.2 ; sys_platform=='darwin'",
             "torchvision==0.8.2+cpu ; sys_platform!='darwin'",
             "torchaudio==0.7.2",
-            "allennlp<2.0.0",
+            "allennlp",
```

Comment:
Because this PR drops the support for AllenNLP < 2.0.0, how about adding the version restriction?
```suggestion
            "allennlp>=2.0.0",
```